src/autocomplete/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment, useState } from 'react';
+import AccessibleAutocomplete from 'accessible-autocomplete/react';
+
+export default function AutoComplete(props) {
+  const [value, setValue] = useState(props.value);
+  const defaultValue = (props.options.find(opt => opt.value === props.value) || {});
+
+  function suggest (query, syncResults) {
+    syncResults(query
+      ? props.options.filter(result => result.label.toLowerCase().indexOf(query.toLowerCase()) !== -1)
```

Comment:
```suggestion
      ? props.options.filter(result => result.label.toLowerCase().includes(query.toLowerCase()))
```

Reply:
yeah I lifted that straight from the example code, it had `var`'s etc :O